config/olm-catalog/ibm-spectrum-scale-csi-operator/2.5.0/ibm-spectrum-scale-csi-operator.v2.5.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,484 @@
+        displayName: nodeMapping
+        path: nodeMapping
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+      - description: tolerations to distribue to IBM Spectrum Scale CSI pods.
+        displayName: tolerations
+        path: tolerations
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+      - description: Node selector for IBM Spectrum Scale CSI Plugin pods.
+        displayName: Plugin Node Selector
+        path: pluginNodeSelector
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+      - description: Key for node selector.
+        displayName: Key
+        path: pluginNodeSelector.key
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:label
+      - description: Value for key.
+        displayName: Value
+        path: pluginNodeSelector.value
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:label
+      - description: Node selector for provisioner sidecar pods.
+        displayName: Provisioner Node Selector
+        path: provisionerNodeSelector
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+      - description: Key for node selector.
+        displayName: Key
+        path: provisionerNodeSelector.key
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:label
+      - description: Value for key.
+        displayName: Value
+        path: provisionerNodeSelector.value
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:label
+      - description: Node selector for attacher sidecar pods.
+        displayName: Attacher Node Selector
+        path: attacherNodeSelector
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+      - description: Key for attacher node selector pods.
+        displayName: Key
+        path: attacherNodeSelector.key
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:label
+      - description: Value for attacher node selector key.
+        displayName: Value
+        path: attacherNodeSelector.value
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:label
+      - description: Node selector for snapshotter sidecar pods.
+        displayName: Snapshotter Node Selector
+        path: clusters
+        path: snapshotterNodeSelector
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+      - description: Key for node selector.
+        displayName: Key
+        path: snapshotterNodeSelector.key
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:label
+      - description: Value for key.
+        displayName: Value
+        path: snapshotterNodeSelector.value
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:label
+      - description: Node selector for resizer sidecar pods.
+        displayName: Resizer Node Selector
+        path: clusters
+        path: resizerNodeSelector
```

Comment:
mutiple `path` mentioned.

Reply:
This was a copy of 2.4.0 CSV which had multiple paths.